tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -295,42 +272,48 @@ private JenkinsPipeline getJenkinsPipeline(String name, int buildId) {
      * @param source    the testing group that verified the tools
      * @param execution the location to test the tools
      */
-    private void handleCreateTests(String api, List<String> source, String execution) {
+    private void handleCreateTests(String api, List<String> source, String execution, String toolname) {
+        if (!execution.equals("jenkins")) {
+            errorMessage("Can only execute on jenkins, no other location is currently supported.  Received " + execution, COMMAND_ERROR);
+        }
+        if (!api.equals("https://www.dockstore.org:8443/api/ga4gh/v1")) {
```

Comment:
This probably shouldn't be hard-coded in case we wish to test against staging for example